templates/base.html

```diff
@@ -77,9 +77,11 @@
         // ask the observer to observe the position of the observerEl - i.e. if it's on/off screen
         observer.observe(observerEl);
       } else {
-        throw new Error(
-          `${selector ? selector : "#navigation"} is not a valid element!`
-        );
+        if (selector) {
+          throw new Error(selector + " is not a valid element!");
```

Comment:
Is throwing necessary? Typically you only want to throw if you're going to catch and handle the error elsewhere. In this case simply using `console.error` should be sufficient.